includes/definitions/apc-epdu.yaml

```diff
@@ -10,3 +10,9 @@ over:
 discovery:
     - sysObjectID:
         - .1.3.6.1.4.1.318.1.3.4.9 
+    - sysObjectID:
+        - .1.3.6.1.4.1.0
```

Comment:
Hmmm okey so this pdu uses a oid that belongs to some trap mib.. .great... would you mind investigate a bit, maybe upgrade firmware if possible? see if they fixed something?

Reply:
I found this `EPDU_MP1_VM_V2813.zip` update, but after reaching technical service, they told me this update was for model EPDU1116M , so it appears to be no other public firmware upgrade.